AgentInterfaces/MaintenanceSchedule.cs

```diff
@@ -16,9 +18,15 @@ public class MaintenanceSchedule
         public IList<MaintenanceEvent> MaintenanceEvents { get; set; }
 
         public string ReportingVmId { get; set; }
+
+        /// <summary>
+        /// The source of the maintenance event. It can be either Azure or PlayFab
+        /// </summary>
+        public string MaintenanceSource { get; set; }
```

Comment:
This field would mark the entire schedule as coming from either Azure or PlayFab--we should just use the existing EventSource field inside the MaintenanceEvent class to mark when an event comes from PlayFab

Reply:
ah, wasn't aware of that field, makes sense, will remove.